packages/grafana-ui/src/components/Select/Select.tsx

```diff
@@ -34,6 +34,7 @@ export interface CommonProps<T> {
   getOptionValue?: (item: SelectOptionItem<T>) => string;
   onChange: (item: SelectOptionItem<T>) => {} | void;
   placeholder?: string;
+  inputId?: string;
```

Comment:
Seems like this property was introduced, but it's not used anywhere. Can u tell what's the reason for introducing it?

Reply:
This is because when the component rendered, React gave the Role select and the Org select fields the same ID which it complained about. This just gives you the option to set your own ID on the field. It's an existing property on the component this is based on.

Reply:
strange, we never seen that error before. If I remove inputId I don't see any error or warning in the chrome console. Where did you see this?

Reply:
@shavonn can you past React's warning/error here?

Reply:
`[DOM] Found 2 elements with non-unique id #react-select-2-input: (More info: https://goo.gl/9p2vKq) <input autocapitalize=​"none" autocomplete=​"off" autocorrect=​"off" id=​"react-select-2-input" spellcheck=​"false" tabindex=​"0" type=​"text" aria-autocomplete=​"list" value style=​"box-sizing:​ content-box;​ width:​ 2px;​ background:​ 0px center;​ border:​ 0px;​ font-size:​ inherit;​ opacity:​ 1;​ outline:​ 0px;​ padding:​ 0px;​ color:​ inherit;​">​ <input autocapitalize=​"none" autocomplete=​"off" autocorrect=​"off" id=​"react-select-2-input" spellcheck=​"false" tabindex=​"0" type=​"text" aria-autocomplete=​"list" value style=​"box-sizing:​ content-box;​ width:​ 2px;​ background:​ 0px center;​ border:​ 0px;​ font-size:​ inherit;​ opacity:​ 1;​ outline:​ 0px;​ padding:​ 0px;​ color:​ inherit;​">​`